src/data/sourcebook-data.ts

```diff
@@ -35,6 +35,7 @@ export class SourcebookData {
 			AncestryData.dwarf,
 			AncestryData.wodeElf,
 			AncestryData.highElf,
+			AncestryData.hakaan,
```

Comment:
I think this is already included in the Orden sourcebook

Reply:
Ah you're right, sorry! That's what I get for skimming it instead of paying attention to the structure of the class. I'll fix that now.